tests/test_glm_synthetic_single_neuron.py

```diff
@@ -15,7 +15,7 @@
     n_basis_funcs=5,
     window_size=ws
 )
-sim_pts = nsl.sample_points.raised_cosine_log(5, ws)
+sim_pts = jnp.clip(neurostatslib.sample_points.raised_cosine_log(5, ws),0,100)
```

Comment:
same comments as `test_glm_synthetic`:
 - why are the lines commented out?
 - this clip shouldn't be necessary, right?

Reply:
as I said before, this will be moved